Added missing include file for release notes (#4723)
magento/devdocs: extensions/amazon-sales/index.md

      composer require magento/services-connector:~1.0.3
      ```

>     - Enter your [authentication keys]({{ site.baseurl }}/install-gde/prereq/connect-auth.html). Your public key is your username; your private key is your password.

Author: ```suggestion
    - Enter your [authentication keys]({{ site.baseurl }}/guides/v2.3/install-gde/prereq/connect-auth.html). Your public key is your username; your private key is your password.
```

Reviewer: this should be `{{ page.baseurl }}`

>     - Enter your [authentication keys]({{ site.baseurl }}/guides/v2.3/install-gde/prereq/connect-auth.html). Your public key is your username; your private key is your password.

Author: ```suggestion
    - Enter your [authentication keys]({{ site.baseurl }}/guides/{{ site.version }}/install-gde/prereq/connect-auth.html). Your public key is your username; your private key is your password.
```

>     - Enter your [authentication keys]({{ site.baseurl }}/guides/{{ site.version }}/install-gde/prereq/connect-auth.html). Your public key is your username; your private key is your password.

Author: ```suggestion
    - Enter your [authentication keys]({{ site.baseurl }}/guides/v{{ site.version }}/install-gde/prereq/connect-auth.html). Your public key is your username; your private key is your password.
```
